src/docs/merge-patch.md

```diff
@@ -73,7 +73,12 @@ Note that this option only works if the patch applies cleanly on the release bra
 1. Modify the commit message:
    1. Prefix the title with "Merged: ".
    1. Remove lines from the footer that correspond to the original CL ("Change-Id", "Reviewed-on", "Reviewed-by", "Commit-Queue", "Cr-Commit-Position"). Definitely keep the "(cherry picked from commit XXX)" line, as this is needed by some tools to relate merges to original CLs.
-   1. Add "No-Try: true", "No-Presubmit: true" and "No-Tree-Checks: true" in the footer.
+   1. Add the following flags to the footer: 
+   ```
```

Comment:
You’ll probably want to indent this more deeply so that it’s correctly nested within the current list item.